admin/Makefile

```diff
@@ -33,7 +33,7 @@ serve: build
 
 ${BUILD_DIR}/venv.timestamp:
 	#Create a Python virtual environment.
-	python3 -m venv ${VENV}
+	virtualenv -p python3 ${VENV}
```

Comment:
Why this change?

Reply:
Because the old version works locally but not on travis, I don't know why.

Reply:
Strange I just do a test...

Reply:
it should work ...
https://travis-ci.org/camptocamp/c2cgeoportal/jobs/293577574